File: src/sdk/android/3/core-classes/memory-storage/geohash/snippets/geohash-1.java

String[] members = {"Palermo", "Catania"};

kuzzle.memoryStorage.geohash("key", members, new ResponseListener<String[]>() {
  @Override
  public void onSuccess(String[] hashes) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});

File: src/sdk/php/3/core-classes/memory-storage/geohash/snippets/geohash-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $hashes = $kuzzle->memoryStorage()->geohash('key', ['Palermo', 'Catania']);
}
catch (ErrorException $e) {

}
